xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -2395,3 +2402,11 @@ void CAMLCodec::SetVideoRate(int videoRate)
   if (am_private)
     am_private->video_rate = videoRate;
 }
+
+int CAMLCodec::GetDecoderVideoRate()
+{
+  struct vdec_status vs;
+  m_dll->codec_get_vdec_state(&am_private->vcodec, &vs);
+  CLog::Log(LOGDEBUG, LOGVIDEO, "CAMLCodec::GetDecoderVideoRate: decoder status: w:%d h:%d f:%d e:%d s:%d", vs.width, vs.height, vs.fps, vs.error_count, vs.status);
```

Comment:
Final one: Pls, remove this log line as it is called for every vsync.

Reply:
Done